blog/2019/06/26/deprecating_official_rex_packages.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Deprecating official Rex packages
+author: Ferenc Erki
+tags: news
+---
+
+As part of our efforts to simplify maintenance around RexOps projects, we have decided to stop building Rex packages on our side. We recommend relying on OS packages maintained by the various distributions in their respective upstream repositories, or to install Rex from CPAN.
+
+---
+
+There are multiple reasons behind this decision, and I'd like to mention a few bigger one below.
+
+## Existing upstream packages in various distributions
+
+Many operating systems are already packaging and shipping Rex in their own respective repositories, so building packages on our own feels like a duplicate effort.
+
+We believe the OS-specific packages in general are also having much better quality than we could achieve on our own. They are maintained by their upstreams who are in much better position to follow changes of the packaging guidelines for the given distribution.
+
+We would like to continue working together with the package maintainers, while we also encourage you to help them out for your own favorite distributions.
+
+We would be happy to link to those official repositories and packages, so please feel free to let us know about them if you think we are not yet aware.
+
+## Collateral packaging burden of dependencies
+
+Packaging Rex also often involves packaging all its dependencies if they are not available in a given distribution. This multiplies the effort needed on our side to provide a proper Rex package not just by having to build all those dependencies, but also tracking each of their own new releases.
+
+We felt this is too far outside the scope of the Rex project, and we wished we could spend the same amount of effort on Rex itself instead.
+
+## Releases are much simpler with CPAN
```

Comment:
maybe refer to `local::lib` for the 'not a super-user' installations?

Reply:
I've slightly rewritten that part to mention the local user's own perl environment, and noted local::lib, perlbrew, etc, by linking to them. Plus added a remark about system wide installation pointing to packages.